spec/stoplight/notifier/logger_spec.rb

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+
+require 'spec_helper'
+
+RSpec.describe Stoplight::Notifier::Logger do
+  it 'is a class' do
+    expect(described_class).to be_a(Class)
+  end
+
+  it 'is a subclass of Base' do
+    expect(described_class).to be < Stoplight::Notifier::Base
+  end
+
+  describe '#formatter' do
+    it 'is initially the default' do
+      expect(described_class.new(nil, nil).formatter)
+        .to eql(Stoplight::Default::FORMATTER)
+    end
+
+    it 'reads the formatter' do
+      formatter = proc {}
+      expect(described_class.new(nil, formatter).formatter)
+        .to eql(formatter)
+    end
+  end
+
+  describe '#logger' do
+    it 'reads the logger' do
+      logger = Logger.new(STDOUT)
```

Comment:
I would prefer testing with [`StringIO`](http://ruby-doc.org/stdlib-2.2.3/libdoc/stringio/rdoc/StringIO.html) instead of standard in and out. You can see some example usage of `StringIO` in [the IO notifier spec](https://github.com/orgsync/stoplight/blob/d195eaadb37a085ff79f7bbd594010cf606b8feb/spec/stoplight/notifier/io_spec.rb#L37).